pkg/tenant/api/v1/client.go

```diff
@@ -52,6 +53,7 @@ type APIv1 struct {
 	endpoint *url.URL
 	client   *http.Client
 	creds    string
+	cookies  []*http.Cookie
```

Comment:
I wasn't sure how to capture the session cookies for the tests (I think we need to be using https to have access to those?) so this is a workaround to ensure that cookies are set. Is there a better way to do this?